be/src/exec/tablet_sink.cpp

```diff
@@ -327,6 +327,9 @@ int NodeChannel::try_send_and_fetch_status() {
 
         _add_batch_closure->reset();
         _add_batch_closure->cntl.set_timeout_ms(_rpc_timeout_ms);
+        if (config::tablet_writer_ignore_eovercrowded) {
+            _add_batch_closure->cntl.ignore_eovercrowded();
```

Comment:
What will happen if OVERCROWDED being ignored?

Reply:
If the controller ignores evorcrowded, this request will be sent even `current_buf > brpc_socket_max_unwritten_bytes`.